shared/my-sites/themes/main.jsx

```diff
@@ -67,7 +71,7 @@ var Themes = React.createClass( {
 	togglePreview: function( theme ) {
 		const site = this.props.sites.getSelectedSite();
 		if ( site.jetpack ) {
-			Action.customize( theme, site );
+			this.props.dispatch( Action.customize( theme, site ) );
```

Comment:
I wonder if it makes sense having the global Redux store as a global variable, so we don't have to pass these props around, if we're not using `connect()`